packages/multichain/src/scope/authorization.test.ts

```diff
@@ -0,0 +1,95 @@
+import { validateAndNormalizeScopes } from './authorization';
+import * as Transform from './transform';
+import type { ExternalScopeObject } from './types';
+import * as Validation from './validation';
+
+jest.mock('./validation', () => ({
+  validateScopes: jest.fn(),
+}));
+const MockValidation = jest.mocked(Validation);
+
+jest.mock('./transform', () => ({
+  normalizeAndMergeScopes: jest.fn(),
+}));
+const MockTransform = jest.mocked(Transform);
+
+const validScopeObject: ExternalScopeObject = {
+  methods: [],
+  notifications: [],
+};
+
+describe('Scope Authorization', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
```

Comment:
This has been happening automatically since https://github.com/MetaMask/core/pull/4417, so it is not needed.

Reply:
https://github.com/MetaMask/core/pull/4784/commits/18d1112b95e756334fbf06d7dd5b9ec24a8783f7